config/webpack.dev.config.js

```diff
@@ -14,27 +14,28 @@ module.exports = {
       publicPath: '/dist/'
    },
    module: {
-      rules: [{
-         test: /\.js$/,
-         exclude: /node_modules/,
-         use: ['react-hot-loader', 'babel-loader']
-      },
-      {
-         test: /\.less$/,
-         use: [
-            'style-loader',
-            'css-loader?modules&importLoaders=1&localIdentName=[path]___[name]__[local]___[hash:base64:5]',
-            'resolve-url-loader',
-            'less-loader'
-         ]
-      }]
+      rules: [
+         {
+            test: /\.js$/,
+            exclude: /node_modules/,
+            use: ['react-hot-loader', 'babel-loader']
+         },
+         {
+            test: /\.less$/,
+            use: [
+               'style-loader',
+               'css-loader?modules&importLoaders=1&localIdentName=[path]___[name]__[local]___[hash:base64:5]',
+               'resolve-url-loader',
+               'less-loader'
+            ]
+         }
+      ]
    },
    resolve: {
-      extensions: ['', '.js']
+      extensions: ['.js'],
+      enforceExtension: false
    },
    plugins: [
-      new webpack.optimize.OccurenceOrderPlugin(),
-      new webpack.HotModuleReplacementPlugin(),
-      new webpack.NoErrorsPlugin()
+      new webpack.HotModuleReplacementPlugin()
```

Comment:
`OccurenceOrderPlugin` is now on by default. Reference: https://webpack.js.org/guides/migrating/#occurrenceorderplugin-is-now-on-by-default